paper.tex

```diff
@@ -208,6 +208,7 @@ \subsection{Refunding}
 To refund or withdraw subscription fee from the smart contract, data provider, broker, and consumer send a transaction to execute the smart contract and are responsible for transaction fee. We assume that only a half of the expected records are uploaded to the MAM channel. For the data provider and broker, they can withdraw a half of total subscription fee from the smart contract and $F_{b}$ \% belongs to the broker while the remaining belongs to the data provider. For consumers, they can get a half refund which should be deducted the transaction fee.
 
 \subsection{Game Theory Evaluation}
+Game Theory is a methodology to discuss strategic interaciton among game players. If we can ensure Nash Equilibrium of decentralized data marketplace exists at certain acceptable range, then we can promise the sustainability of decentralized data marketplace. Fan Liang et al\cite{SurveyBigDataPricing} listed several different types of game theory models. To describe the scenario more precisely, we decided to use repeated game to build our game theory model which meets the scenario of subscription economy more.
```

Comment:
You don't have to mention Fan Liang. Instead, you just summarize the work within descriptions and append the citation.